spec/smoke

```diff
@@ -97,11 +97,24 @@ main() {
   rails runner "Search.refresh" || exit 1
   echo "[success]"
 
+  echo "add indexes to materialized view"
+  rails runner "ActiveRecord::Base.connection.execute 'CREATE INDEX searches_test_1 ON searches USING btree (results);'" || exit 1
+  rails runner "ActiveRecord::Base.connection.execute 'CREATE INDEX searches_test_2 ON searches USING btree (user_id);'" || exit 1
+  echo "[success]"
+
+  echo "update materialized view"
+  rails generate scenic:view search --materialized
+  echo "SELECT 'test'::text AS results" > db/views/searches_v02.sql
+  rake db:migrate
+  verifySearchResults 'test'
+
   echo "rake db:rollback"
   rake db:rollback
+  rake db:rollback
   echo "[success]"
 
-  echo "rails destroy scenic:view search --materialized"
+  echo "rails destroy scenic:model search --materialized"
```

Comment:
What we're actually testing here is the `model` destroy since we never tested that previously. We have to first do a `view` destroy so that the second version of the view is deleted.